documentation/Functions.md

```diff
@@ -278,6 +286,8 @@ port: Port on which the server is running
 path: The URL path on which is checked if the server is running
 
 If the tutorial should be tested on the console environment, you have to specify a port.
+
+Note: To execute this function, devonfw-ide must be installed.
```

Comment:
For this command the devonfw-ide is not necessarily needed. Maven has to be installed. So if someone installs Maven before (e.g. with a setup script), then the command should also work